### PR TITLE
Update mac-updates.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the correct channel name

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10866

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/mac-updates.md [hunk 1]
```diff
@@ -65,9 +65,9 @@ The `Current` channel contains the most stable version of the product.
 > [!IMPORTANT]
 > Prior to Microsoft AutoUpdate version 4.29, channels had different names:
 >
-> - `Beta` was named `InsiderFast` (Insider Fast)
-> - `Preview` was named `External` (Insider Slow)
-> - `Current` was named `Production`
+> - `Beta Channel` was named `InsiderFast` (Insider Fast)
+> - `Current channel` (Preview) was named `External` (Insider Slow)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-updates.md]:
```suggestion
> - `Current Channel` (Preview) was named `External` (Insider Slow)
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder

**Pull request merged**